contribs/gmf/less/vars.less

```diff
@@ -22,7 +22,7 @@
 @above-search-index: 6;
 
 
-@nav-width: 26rem;
+@nav-width: 400px;
```

Comment:
I don't agree, px should be avoid as possible, and rem should be used every where is possible, with that we can easy do a choerant zoom all the application by only changing the root font!